packages/e2e-tests/config/setup-test-framework.js

```diff
@@ -240,6 +240,9 @@ beforeAll( async () => {
 	await trashAllPosts( 'wp_block' );
 	await setupBrowser();
 	await activatePlugin( 'gutenberg-test-plugin-disables-the-css-animations' );
```

Comment:
I think this means we can remove that plugin no?

Reply:
Yep, but I'd rather keep it for a while though. As mentioned in the original issue, I don't think we're quite there yet to disable this plugin entirely. Disabling it now would probably slow down the tests a little too.